e2e/features/edit-appeal.feature

```diff
@@ -0,0 +1,80 @@
+Feature: Edit appeal application
+
+  Background:
+    Given I am signed in as a `Legal Rep`
+    And I create a new case
+    And I save my initial appeal
+
+  @edit-appeal @RIA-653
+  Scenario: Edit a draft appeal
+
+    When I select the `Edit appeal` Next step
+    Then I am on the `Edit appeal` page
+
+    Given I am on the `Home Office reference` page
+    When I type `B123456/999` for the Home Office reference number field
+    And I type `31-12-2018` for the `Date on the decision letter` field
```

Comment:
Is it worth trying an Edit scenario where we input an invalid date or reference number?

Reply:
Yeah we can do that -- do you want to extend the test case?

Reply:
Sure I can do that.